docs/products/opensearch/dashboards/howto/opensearch-alerting-dashboard.rst

```diff
@@ -0,0 +1,124 @@
+Create alerts with OpenSearch Dashboards
+========================================
+
+OpenSearch alerting feature sends notifications when data from one or more indices meets certain conditions that can be customized.
+Use case examples are such as monitoring for HTTP status code 503, CPU load average above certain percentage or watch for counts of a specific keyword in logs for a specific amount of interval,
+notification to be configured to be sent via email, slack or custom webhooks and other destination, in this example we are using slack as the destination.
+
+In the following example, we are using a ``sample-host-health`` index as datasource to create a simple alert to check cpu load, action will be triggered when average of ``cpu_usage_percentage`` over ``3`` minutes is above ``75%``
+
+Create using Dashboards UI
+**************************
+
+In order to create an alert via OpenSearch Dashboards interface, follow these steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io>`_ and select your OpenSearch service.
+
+2. Click the **Overview** tab -> **OpenSearch Dashboards** under `Connection Information`
+
+   This opens the OpenSearch Dashboard
+
+3. In OpenSearch Dashboard Click  on top left to open the left side panel -> Under `OpenSearch Plugins` -> **Alerting**
+
+
+To configure each alert the following needs to be created, we will walk-through configuration of each section.
+
+- ``Destination``
+- ``Monitor``
+- ``Data source``
+- ``Query``
+- ``Trigger``
+
+Create a destination
+********************
+Destination is a location for notifications to be delivered when an action is triggered.
+
+1. Click the **Destination** tab -> **Add destination**
+   
+2. **Name** -> ``slack-test``
```

Comment:
Fill in the fields: **Name** - "slack-test", **Type** - "Slack",  **Webhook URL:** - "https://your_slack_webhook_URL".